### PR TITLE
Handle mastodon links :mammoth: 

### PR DESCRIPTION
**Pull request by davidwilby**
Relates to https://github.com/GateNLP/we-verify-app-assistant/issues/50 and https://github.com/GateNLP/we-verify-app-assistant/pull/51

Requires merge and deployment of https://github.com/GateNLP/we-verify-app-assistant/pull/51

@Sallaa please check the base and target branches for this branch (both `master`) - the branch structure of this project is confusing++.

### CODE REVIEW

File: src/redux/sagas/assistantSaga.jsx [hunk 1]
```diff
@@ -459,7 +459,7 @@ function formatTelegramLink(url) {
   let urlType = matchPattern(url, KNOWN_LINK_PATTERNS);
   if (urlType !== KNOWN_LINKS.TELEGRAM) {
     throw new Error(
-      "formatTelegramLink: Expected telegram link but got " + urlType
+      "formatTelegramLink: Expected telegram link but got " + urlType,
```
Comment (twinkarma) [inline src/redux/sagas/assistantSaga.jsx:462]: Why the additional comma?

Comment (davidwilby) [reply src/redux/sagas/assistantSaga.jsx:462]: I think this must be from the linter.

Comment (davidwilby) [reply src/redux/sagas/assistantSaga.jsx:462]: Can remove if required.

Comment (twinkarma) [reply src/redux/sagas/assistantSaga.jsx:462]: Shouldn't be much of a difference either way so I'll leave it up to you. The rest is looking good.

**Review by twinkarma — APPROVED** [review]
LGTM!

**Pull request merged**